pkgs/development/tools/packer/default.nix

```diff
@@ -40,7 +44,178 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    mv bin/* $out/bin
+    mv bin/* $out/bin # */
+  '';
+
+  passthru.updateScript = writeScript "update-packer" ''
```

Comment:
probably it would be good to have this in a separate file. but i understand why it is done inline, it is just easier to iterate. i would say it is up to a maintainer of the package to decide what is easier for them.